Sysvale/component-library
# [WIP] feature/improve-stepper-usability

#### What is the purpose of this pull request?
To make the active step more clear
To make the click event that change the active step customizable

#### How should this be manually tested?
Change the prop 'changeStepOnClick' on the knobs that and see it changes the behaviour of the 
click on the steps
See if the ui change that makes the active step more clear actually does it

#### Types of changes
